vacationtime-solution/README.md

```diff
@@ -0,0 +1,25 @@
+# Manage employee vacation/time off requests
+
+Allows employers to easily view and manage their employees' requests for vacation time/time off. If approved, a shared Google Calendar event will be created with the corresponding vacation start & end times. If not approved, employees will be notified via email. 
```

Comment:
This should be a one sentence summary, you can look at a previous example for the format of this file:
https://raw.githubusercontent.com/gsuitedevs/solutions/master/vacation-calendar/README.md

also add a last updated